src/charm.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env -S LD_LIBRARY_PATH=binlibs python3
```

Comment:
This is magical, but seems to also work in focal. Alternatively, we can try to override the hooks to also add this, but I'm not sure there is a clean way to do that either.

Reply:
IMHO, it's cleaner to keep it here.